torch/csrc/distributed/rpc/tensorpipe_utils.cpp

```diff
@@ -1,10 +1,12 @@
+#include <torch/csrc/distributed/rpc/macros.h>
 #include <torch/csrc/distributed/rpc/tensorpipe_utils.h>
```

Comment:
usually the header of the cpp should be the first include?

Reply:
addressed

Reply:
The header of the cpp should come first _in its own block_ (i.e., followed by a blank line), in which case clang-format will respect that.